lib/rucio/web/rest/flaskapi/v1/replicas.py

```diff
@@ -883,6 +884,57 @@ def post(self):
         return 'Created', 201
 
 
+class QuarantineReplicas(ErrorHandlingMethodView):
+
+    def post(self: object) -> object:
+        """
+        ---
+        summary: Quarantine replicas.
+        description: Quarantine replicas.
+        tags:
+          - Data Identifiers
+        requestBody:
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  description: replica dids
+        responses:
+          201:
+            description: Quarantined
+          400:
+            description: InputValidationError
+          401:
+            description: AccessDenied
+          404:
+            description: RSENotFound
+          503:
+            description: ResourceTemporaryUnavailable
```

Comment:
Good job, however I'd like this part to be a bit more specific. The API documentation should be verbose on its own, since we should not force users to look into the code to figure out what the input and output of the endpoint is.

Something like
```text
requestBody:
  content:
    application/json:
      schema:
        type: object
        properties:
          rse:
            description: The name of the rse.
            type: string
          rse_id:
            description: The id of the rse.
            type: string
          replicas:
            description: The replicas to quarantine.
            type: array
            items:
              type: object
              properties:
                scope:
                  description: The scope of the replica.
                  type: string
                name:
                  description: The name of the replica.
                  type: string
                path:
                  description: The path to the replica.
                  type: string
```